src/app/client/src/app/modules/core/services/course/course.service.ts

```diff
@@ -17,6 +17,7 @@ import { HttpClient } from '@angular/common/http';
 })
 export class CoursesService {
   private enrolledCourses: Array<ICourses>;
+  private recommendedCourses: Array<any>;
```

Comment:
Any reason to remove ICourses,  you could have updated the properties in it, this is core services , not recommend to change

Reply:
No, Santhosh 
Mistakenly changed to any.